src/contracts/libraries/SafetyBits.sol

```diff
@@ -1,53 +1,51 @@
 //SPDX-License-Identifier: BUSL-1.1
 pragma solidity 0.8.22;
 
-import "../types/LockTypes.sol";
-
-// TODO remove
-//import {TestUtils} from "../../../test/base/TestUtils.sol";
-// import "forge-std/Test.sol";
+import "src/contracts/types/LockTypes.sol";
 
 // uint16 bit layout:  BBBB BBBB AAAA
 // Where A = BaseLock, B = ExecutionPhase,
 
 uint16 constant EXECUTION_PHASE_OFFSET = uint16(type(BaseLock).max) + 1;
 
 // NOTE: No user transfers allowed during HandlingPayments
-uint16 constant SAFE_USER_TRANSFER = 0x0ae0; // 2784
-    /* 
-        uint16( 1 << (EXECUTION_PHASE_OFFSET + uint16(ExecutionPhase.PreOps))
-            | 1 << (EXECUTION_PHASE_OFFSET + uint16(ExecutionPhase.UserOperation))
-            | 1 << (EXECUTION_PHASE_OFFSET + uint16(ExecutionPhase.PreSolver))
-            | 1 << (EXECUTION_PHASE_OFFSET + uint16(ExecutionPhase.PostSolver))
-            | 1 << (EXECUTION_PHASE_OFFSET + uint16(ExecutionPhase.PostOps))
-        */
+uint16 constant SAFE_USER_TRANSFER = uint16(
+    1 << (EXECUTION_PHASE_OFFSET + uint16(ExecutionPhase.PreOps))
+        | 1 << (EXECUTION_PHASE_OFFSET + uint16(ExecutionPhase.UserOperation))
+        | 1 << (EXECUTION_PHASE_OFFSET + uint16(ExecutionPhase.PreSolver))
+        | 1 << (EXECUTION_PHASE_OFFSET + uint16(ExecutionPhase.PostSolver))
+        | 1 << (EXECUTION_PHASE_OFFSET + uint16(ExecutionPhase.PostOps))
+);
 
 // NOTE: No Dapp transfers allowed during UserOperation
-uint16 constant SAFE_DAPP_TRANSFER = 0x0ca0; // 3232
-    /*
-        uint16(1 << (EXECUTION_PHASE_OFFSET + uint16(ExecutionPhase.PreOps))
-            | 1 << (EXECUTION_PHASE_OFFSET + uint16(ExecutionPhase.PreSolver))
-            | 1 << (EXECUTION_PHASE_OFFSET + uint16(ExecutionPhase.HandlingPayments))
-            | 1 << (EXECUTION_PHASE_OFFSET + uint16(ExecutionPhase.PostOps))
-        */
+uint16 constant SAFE_DAPP_TRANSFER = uint16(
+    1 << (EXECUTION_PHASE_OFFSET + uint16(ExecutionPhase.PreOps))
+        | 1 << (EXECUTION_PHASE_OFFSET + uint16(ExecutionPhase.PreSolver))
+        | 1 << (EXECUTION_PHASE_OFFSET + uint16(ExecutionPhase.HandlingPayments))
+        | 1 << (EXECUTION_PHASE_OFFSET + uint16(ExecutionPhase.PostOps))
+);
```

Comment:
Permit69 token transfers allowed from user but not from dapp in `ExecutionPhase.PostSolver`. Is this intentional?

Reply:
no, good catch.

Reply:
313d0c7